src/manipulation/wrapMap.js

```diff
@@ -3,12 +3,11 @@ define( function() {
 // We have to close these tags to support XHTML (#13200)
 var wrapMap = {
 
-	// Support: IE9
```

Comment:
I removed the `IE9` mention as I believe it's not an IE9 issue but just a general need for wrapping. This comment was added by you, @markelog, could you elaborate?